### PR TITLE
Added mx show-app-version and set-app-version documentation

### PR DESCRIPTION
**Pull request by mikita-haurykau**
Documentation about mx show-app-version and set-app-version

### CODE REVIEW

File: content/en/docs/refguide/general/mx-command-line-tool.md [hunk 1]
```diff
@@ -256,6 +256,129 @@ If there are conflicts during the merge, you have to resolve those by opening th
 | 3           | This code means an exception – an error occurred during the merge. Error details are printed to the command line output. |
 | 4           | The version is unsupported.                                  |
 
-### 3.6 Undocumented Options
+### 3.6 mx show-app-version Command [version 9.24.2+]
+
+The mx show-app-version command allows you to see the version of your solution ([publisher side](/appstore/creating-content/sol-solutions-guide)) and version of a solution package your app is based on ([consumer side](/appstore/creating-content/sol-solutions-impl))
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/mx-command-line-tool.md]: @Adam-Dupaski , to align with Creating Content guide, shall I change "publisher side" to something like "the version of the solution that you develop"?

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/mx-command-line-tool.md]: Similar for "consumer side", shall I change it to "the version of a solution package your app is based on when you consumed a solution"?

Comment (Adam-Dupaski) [reply content/en/docs/refguide/general/mx-command-line-tool.md]: Both "publisher" and "consumer" are used in the Creating Content Guide, so I think they could be used here. But for more clarity, maybe combine both approaches?:

The `mx show-app-version` command allows you to see the following:
* The [publisher-side](/appstore/creating-content/sol-solutions-guide) version of your solution (meaning, the version of the solution that you develop)
* The [consumer-side](/appstore/creating-content/sol-solutions-impl) version of the solution package that your app is based on (meaning, the version of the solution package when you consumed the solution)

**Comment by MariaShaposhnikova** [comment]
@mikita-haurykau , I've reviewed the PR, please double-check that ti is technically correct.

**Comment by MariaShaposhnikova** [comment]
I've recreated and merged this PR in this one: https://github.com/mendix/docs/pull/5959, because a new major version was introduced and it was easier to reproduce it.

**Pull request closed without merging**